src/tv2-common/migrations/index.ts

```diff
@@ -173,6 +174,74 @@ export function mapGfxTemplateToDesignTemplateAndDeleteOriginals(
 	})
 }
 
+export function moveSelectedGfxSetupNameToGfxDefaults(versionStr: string, fromValue: string, targetTable: string) {
+	return literal<MigrationStepShowStyle>({
+		id: `${versionStr}.moveSelectedGfxSetupName.${fromValue}.ToTable.${targetTable}`,
+		version: versionStr,
+		canBeRunAutomatically: true,
+		validate: (context: MigrationContextShowStyle) => {
+			const singleValue = (context.getBaseConfig(fromValue) as unknown) as string
+			const designatedTable = (context.getBaseConfig(targetTable) as unknown) as TableConfigItemValue | undefined
+
+			if (!singleValue || !Array.isArray(designatedTable)) {
+				return false
+			}
+
+			return singleValue
+		},
+		migrate: (context: MigrationContextShowStyle) => {
+			const singleValue = context.getBaseConfig(fromValue) as BasicConfigItemValue
+			const designatedTable = (context.getBaseConfig(targetTable) as unknown) as TableConfigItemValue
+			const setupName = 'DefaultSetupName'
+
+			designatedTable[0][setupName] = singleValue
+
+			context.setBaseConfig(targetTable, (designatedTable as unknown) as ConfigItemValue)
+			context.removeBaseConfig(fromValue)
+		}
+	})
+}
+
+export function moveSelectedGfxSetupNameToGfxDefaultsInVariants(
+	versionStr: string,
+	fromValue: string,
+	targetTable: string
```

Comment:
We've probably named it like this in multiple places, but I feel like `targetTableName` or rather `targetTableId` would be a better name here

Reply:
Looking at the name `moveSelectedGfxSetupNameToGfxDefaultsInVariants` it doesn't really make sense to have `fromValue` and `targetTable` at all. It allows me to call it like this `moveSelectedGfxSetupNameToGfxDefaultsInVariants('1.8.3', 'CasparCGLodaingClip', 'GfxTemplates')` which would then do something entirely different than what the name implies.

The good thing about this very specific method name is that we don't have to make it "dynamic" by having parameters such as `fromValue` and `targetTable`